tests/page-objects/default/login/login.wdio.page.js

```diff
@@ -17,7 +17,8 @@ const getErrorMessage = async () => {
   return await (await errorMessageField()).getText();
 };
 
-const login = async ({ username, password, createUser = false, locale, loadPage = true, privacyPolicy, adminApp }) => {
+// eslint-disable-next-line max-len
+const login = async ({ username, password, createUser = false, locale, loadPage = true, privacyPolicy, adminApp }, isOldNav = false) => {
```

Comment:
We try as much as possible not to disable the lint. Once you remove the `isOldNav`, then the lint will pass fine.